### PR TITLE
ENH: Reduce VTK wheel size

### PR DESCRIPTION
**Pull request by larsoner**
These extra options reduce the size from 522 MB to 108 MB. For reference the 9.0.1 PyPi manylinux wheels are 103 MB.

We could update the second, off-screen command but it gets a bit wordy. I think it's enough to have these in one place and people can copy-paste the parts they need from each.

### CODE REVIEW

**Review by banesullivan — APPROVED** [review]

**Comment by akaszynski** [comment]
This is great @larsoner!  Thanks!
